index.html

```diff
@@ -1406,6 +1406,12 @@ <h4>Text</h4>
             is indicated using the <a>Text Language Source</a> property.</p>
           <p>If the <a>Text</a> object, or part of it, represents something more specific, or different,
             to its parent <a>Script Event</a>, it can have a different <a>Represents</a> property.</p>
+          <p>The <a>Text</a> object, and every one of its parts, MUST have a valid <a>Represents</a> property.
+            The presence of the property itself is considered optional
+            because if the <a>Text</a> object omits the
+            property then the property is inherited from the parent <a>Script Event</a>.
```

Comment:
I'm not sure I understand why we need these additional sentences. 
Regarding the first sentence, any use of any property must use a valid value. 
Regarding the second, it is already covered by the notion of the computed value.

Reply:
The text does not mention the computed value though.

The reason I added this text is to ensure that it is clear that sub-parts of a Text need to have a valid `daptm:represents`.

For example:

```xml
<p daptm:represents="x-valid">This is okay</p>
```

but is this okay?

```xml
<p daptm:represents="x-valid">Text with a <span daptm:represents="invalid">sub-part</span> that needs to be not okay</p>
```

In the second example, the Text (the outer `<p>`) has a valid `daptm:represents` but also contains a `<span>` that does not. Without this additional spec text I think it is not clear if that is valid.

Reply:
ok